apps/stripe/src/app/api/saleor/transaction-initialize-session/response-params.ts

```diff
@@ -0,0 +1,22 @@
+import { z } from "zod";
+
+export const responseData = z
+  .object({
+    stripeClientSecret: z
+      .string({
+        required_error: "Client secret is required",
+      })
+      .min(1),
+  })
+  .brand("TransactionIntializeSessionResponseData");
+
+export type ResponseDataType = z.infer<typeof responseData>;
+
+export const stripePaymentIntentId = z
+  .string({
+    required_error: "Payment intent id is required",
+  })
+  .startsWith("pi_")
+  .brand("StripePaymentIntentId");
```

Comment:
This is not something specific to response. Response is actually json (that leaves the app) + class that creates such json

On the other hand StripePaymentIntentId is part of our domain, so I think it should be in modules/stripe just like other VOs